deploy-manage/deploy/cloud-enterprise.md

```diff
@@ -1,27 +1,56 @@
+Refer to [](./cloud-enterprise/ece-overview.md) for a detailed introduction to ECE, including its features, use cases, and architecture.
 
-% Scope notes: Ensure the landing page makes sense and its aligned with the section overview and the overview about orchestators. What content should be in deployment types overview or in the main overview and what in the ECE landing page...
+::::{tip}
+If you are looking for a solution to orchestrate and manage {{es}} clusters natively on Kubernetes, consider using [Elastic Cloud on Kubernetes (ECK)](./cloud-on-k8s.md) instead of ECE. ECK enables you to orchestrate Elastic Stack applications seamlessly on Kubernetes, leveraging it as the underlying platform for deployment, scaling, and lifecycle management.
+::::
 
-% Use migrated content from existing pages that map to this page:
+% should we use a L2 heading here or just continue?
+## Section overview
 
-% - [ ] ./raw-migrated-files/cloud/cloud-enterprise/Elastic-Cloud-Enterprise-overview.md
-%      Notes: 2 child docs
-% - [ ] ./raw-migrated-files/cloud/cloud-enterprise/ece-administering-ece.md
-%      Notes: redirect only
+This section focuses on deploying ECE and the orchestrating and configuring {{es}} clusters, also referred to as `deployments`.
 
-⚠️ **This page is a work in progress.** ⚠️
+In ECE, a deployment is a managed {{stack}} environment that provides users with an {{es}} cluster along with supporting components such as {{kib}} and other optional services like APM and Fleet.
 
-The documentation team is working to combine content pulled from the following pages:
+This section covers the following tasks:
 
-* [/raw-migrated-files/cloud/cloud-enterprise/Elastic-Cloud-Enterprise-overview.md](/raw-migrated-files/cloud/cloud-enterprise/Elastic-Cloud-Enterprise-overview.md)
-* [/raw-migrated-files/cloud/cloud-enterprise/ece-administering-ece.md](/raw-migrated-files/cloud/cloud-enterprise/ece-administering-ece.md)
\ No newline at end of file
+* [Deploy ECE](./cloud-enterprise/deploy-an-orchestrator.md)
+    - [Prepare the environment](./cloud-enterprise/prepare-environment.md)
+    - [Install ECE](./cloud-enterprise/install.md)
+    - [Air gapped installations](./cloud-enterprise/air-gapped-install.md)
+    - [Configure ECE](./cloud-enterprise/configure.md)
+
+* [Work with deployments](./cloud-enterprise/working-with-deployments.md)
+  - Use [](./cloud-enterprise/deployment-templates.md) to [](./cloud-enterprise/create-deployment.md)
+  - [](./cloud-enterprise/customize-deployment.md)
+  - Use the deployment [Cloud ID](./cloud-enterprise/find-cloud-id.md) and [Endpoint URLs](./cloud-enterprise/find-endpoint-url.md) for clients connection
+
+* Learn how to use the multiple [](./cloud-enterprise/tools-apis.md) available
+
+Other sections of the documentation also include important tasks related to ECE:
+
+* [Secure your ECE installation](../security/secure-your-elastic-cloud-enterprise-installation.md)
```

Comment:
technically, the rest of the security section might be relevant as well (e.g. securing settings, saved objects?) 

could call out the ece section specifically though

Reply:
Secure your ECE installation was a page in the original content with a lot of security related topics for ECE as chid docs. I hope to have something similar in the security section there.

It's true that I expect that to be focused on the orchestrator security and not on deployments security.

I'll add a link to the "secure your deployments" area which was  https://www.elastic.co/guide/en/cloud-enterprise/current/ece-securing-clusters.html in the original docs. Thanks!

What I wouldn't do linking in the main page of ECE to something as specific as `kibana saved objects`, but if you share other examples that I might be missing please share!

For `secure settings` I expect them to be part of the security content mentioned previously.